api/webserver.go

```diff
@@ -0,0 +1,16 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/gin-gonic/gin"
+)
+
+func RunWebServer(port *int) error {
```

Comment:
If we change the way we read port from terminal and change it to int, we need to get the refference of that variable to this function.
This function doesn't need to care about how we read the port from anywhere! the caller shoud care about how to prepare requirements(arguments) for this function.

Reply:
Ok,done